common/fixation_entry_test.go

```diff
@@ -66,7 +66,7 @@ func testWithTemplate(t *testing.T, playbook []template, countObj int, countVS i
 	var dummy sdk.Coin
 
 	for i := 0; i < countObj; i++ {
-		coins = append(coins, sdk.Coin{Denom: "utest", Amount: sdk.NewInt(int64(i+1))})
```

Comment:
All changes to this file are lint fixes unrelated to this PR. Please separate into a separate one.
(Also, note that currently linting of test files is off in the lint config).

Reply:
reverted the changes. I put a task in the backlog to enable lint check for all test files and fix lint issues: [CNS-379](https://lavanet.atlassian.net/browse/CNS-379)